docs/aspace_version_control/readme.md

```diff
@@ -5,10 +5,15 @@ The [get_eads](https://github.com/pulibrary/lib_jobs/blob/main/lib/tasks/lib_job
 ). It is [scheduled](https://github.com/pulibrary/lib_jobs/blob/main/config/schedule.rb) to run on the first day of the month.
 
 ### Steps
-1. authenticate to GitLab and ASpace
-2. create directory structure if it doesn't already exist:
+1. authenticates to GitLab and ASpace
+2. creates directory structure if it doesn't already exist:
    
    The GitLab directory structure organizes the EAD files into directories named after the holding repository and, in the case of Mudd only (for legacy reasons), the holding library. Saving the files into the same structure on download allows the process to commit all files at once.
-4. download EADs for all repos
-6. add and commit to GitLab
+4. downloads EADs for all repos
+6. adds and commits to GitLab
+
+### To Run Manually
+1. ssh into lib-jobs-prod2 (don't forget VPN)
```

Comment:
can we replace 1. with :

`ssh deploy@lib-jobs-prod2.princeton.edu`

Reply:
done! thanks @christinach !